pkgs/misc/vim-plugins/default.nix

```diff
@@ -941,6 +942,35 @@ rec {
 
   };
 
+  color_coded = buildVimPluginFrom2Nix { # created by nix#NixDerivation
+    name = "color_coded-2017-10-30";
+    src = fetchgit {
+      url = "https://github.com/jeaye/color_coded";
+      rev = "b450a19bdaaced760ad4c78558043d77bf3ed902";
+      sha256 = "1sq68c2hx4q95ibdxkywbdksc6lcmmb206acy0704957m8sc8n1f";
+    };
+    dependencies = [];
+    buildInputs = [
+      stdenv
```

Comment:
you don't need to add stdenv here, also nativeBuildInputs is more correct for cmake.

Reply:
I'll remove stdenv, move cmake to nativeBuildInputs, and squash the commits tonight.

Reply:
Did this ever happen?